Cannot find module 'windicss/colors' in vscode

Every time I open VS Code I get this message. Tried by reinstalling windicss extension but didn't work. 

OS: Ubuntu 20.10 64 Bit
Vscode: v1.54.3
node: 12.18.3

`ERROR: Error: Cannot find module 'windicss/colors'
Require stack:
- /home/projects/tailwind.config.cjs
- /home/.vscode/extensions/voorjaar.windicss-intellisense-0.7.2/out/extension.js
- /usr/share/code/resources/app/out/vs/loader.js
- /usr/share/code/resources/app/out/bootstrap-amd.js
- /usr/share/code/resources/app/out/bootstrap-fork.js
Error: Cannot find module 'windicss/colors'
Require stack:
- /home/projects/SLNES/UI_test/tailwind.config.cjs
- /home/dhanvi/.vscode/extensions/voorjaar.windicss-intellisense-0.7.2/out/extension.js
- /usr/share/code/resources/app/out/vs/loader.js
- /usr/share/code/resources/app/out/bootstrap-amd.js
- /usr/share/code/resources/app/out/bootstrap-fork.js
	at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1019:15)
	at Module._load (internal/modules/cjs/loader.js:895:27)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12738)
	at Function.s._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:27167)
	at Function._._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:23919)
	at Function.W._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:96:4637)
	at Module.require (internal/modules/cjs/loader.js:1079:19)
	at v (/usr/share/code/resources/app/out/vs/loader.js:3:12287)
	at Object.<anonymous> (/home/projects/SLNES/UI_test/tailwind.config.cjs:1:78)
	at Module.u._compile (/usr/share/code/resources/app/out/vs/loader.js:3:12841)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
	at Module.load (internal/modules/cjs/loader.js:1039:32)
	at Module._load (internal/modules/cjs/loader.js:932:14)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12738)
	at Function.s._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:27167)
	at Function._._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:23919)
	at Function.W._load (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:96:4637)
	at Module.require (internal/modules/cjs/loader.js:1079:19)
	at v (/usr/share/code/resources/app/out/vs/loader.js:3:12287)
	at /home/dhanvi/.vscode/extensions/voorjaar.windicss-intellisense-0.7.2/out/extension.js:26358:38
	at step (/home/dhanvi/.vscode/extensions/voorjaar.windicss-intellisense-0.7.2/out/extension.js:62:23)
	at Object.next (/home/dhanvi/.vscode/extensions/voorjaar.windicss-intellisense-0.7.2/out/extension.js:43:53)
	at fulfilled (/home/dhanvi/.vscode/extensions/voorjaar.windicss-intellisense-0.7.2/out/extension.js:33:58)
	at processTicksAndRejections (internal/process/task_queues.js:97:5)
Windi CSS Intellisense is now active!`

--- a/package.json
+++ b/package.json
@@ -223,5 +223,8 @@
     "typescript": "^4.2.3",
     "windicss": "^2.5.11",
     "windicss-analysis": "^0.1.2"
+  },
+  "dependencies": {
+    "@windicss/plugin-utils": "^0.12.5"
   }
 }

--- a/package.json
+++ b/package.json
@@ -221,10 +221,10 @@
     "ts-jest": "^26.5.4",
     "tslib": "^2.1.0",
     "typescript": "^4.2.3",
-    "windicss": "^2.5.11",
-    "windicss-analysis": "^0.1.2"
+    "windicss": "^2.5.14",
+    "windicss-analysis": "^0.1.3"
   },
   "dependencies": {
-    "@windicss/plugin-utils": "^0.12.5"
+    "@windicss/plugin-utils": "^0.14.0"
   }
 }

--- a/src/lib/core.ts
+++ b/src/lib/core.ts
@@ -1,12 +1,11 @@
 import { workspace } from 'vscode';
-import { resolve } from 'path';
 import { Processor } from 'windicss/lib';
 import { flatColors } from 'windicss/utils';
 import { hex2RGB, highlightCSS } from '../utils';
 import { utilities as dynamic, negative } from '../utils/utilities';
-import { registerTS } from 'sucrase/dist/register';
 import type { Core } from '../interfaces';
 import { Log } from '../utils/Log';
+import { loadConfiguration } from "@windicss/plugin-utils";
 
 export async function init(): Promise<Core> {
   try {
@@ -15,16 +14,11 @@ export async function init(): Promise<Core> {
     let config;
     if (files[0]) {
       configFile = files[0].fsPath;
-      if (configFile.endsWith('.ts')) {
-        registerTS();
-        delete require.cache[require.resolve(configFile)];
-        const mod = require(configFile);
-        if (mod.default) config = mod.default;
-      } else {
-        delete require.cache[require.resolve(resolve(configFile))];
-        config = import(resolve(configFile));
-      }
+      const { error, resolved } = loadConfiguration({ config: configFile })
+      if (error) throw error
+      if (!resolved) throw new Error("undefined error")
       Log.info(`Loading Config File: ${configFile}`);
+      config = resolved
     }
     const processor = new Processor(config);
     const separator = processor.config('separator', ':') as string;

--- a/src/lib/core.ts
+++ b/src/lib/core.ts
@@ -14,7 +14,7 @@ export async function init(): Promise<Core> {
     let config;
     if (files[0]) {
       configFile = files[0].fsPath;
-      const { error, resolved } = loadConfiguration({ config: configFile })
+      const { error, resolved } = await loadConfiguration({ config: configFile })
       if (error) throw error
       if (!resolved) throw new Error("undefined error")
       Log.info(`Loading Config File: ${configFile}`);

